README.md

```diff
@@ -0,0 +1,85 @@
+![OpenWrt logo](/logo.svg)
+
+OpenWrt Project is a Linux operating system targeting embedded devices. Instead
+of trying to create a single, static firmware, OpenWrt provides a fully
+writable filesystem with package management. This frees you from the
+application selection and configuration provided by the vendor and allows you
+to customize the device through the use of packages to suit any application.
+For developers, OpenWrt is the framework to build an application without having
+to build a complete firmware around it; for users this means the ability for
+full customization, to use the device in ways never envisioned. 
+
+Latest release: **19.07.3**
+
+Sunshine!
+
+## Development
+
+To build your own firmware you need a Linux, BSD or MacOSX system (case
+sensitive filesystem required). Cygwin is unsupported because of the lack of a
+case sensitive file system.
+
+### Requirements
+
+You need the following tools to compile Openwrt, the package name vary between
+distributions.
+
+### Quickstart
+
+    gcc binutils bzip2 flex python3 perl make find grep diff unzip gawk getopt
+    subversion libz-dev libc-dev
```

Comment:
Seems like the "You need the following tools ..." and "Requirements" headline should be placed before that block, instead of "Quickstart"?

Reply:
I moved the block up as initially intended.